src/olympia/editors/helpers.py

```diff
@@ -474,11 +474,12 @@ def get_actions(self, request):
                 }
         if self.version:
             actions['info'] = {
```

Comment:
Change the action name as well ? (It probably breaks a few tests but it'd be more coherent)